gcn.tf

```diff
@@ -3,7 +3,7 @@
 module "network" {
   count                   = var.use_existing_network ? 0 : 1
   source                  = "terraform-google-modules/network/google"
-  version                 = "9.3.0"
+  version                 = "10.0.0"
```

Comment:
Lets take a careful look at this. Its a major version upgrade - we might have to test this thoroughly